apps/examples/webserver_test/README.txt

```diff
@@ -0,0 +1,18 @@
+examples/webserver_test
+^^^^^^^^^^^^^^^^^^
```

Comment:
Thanks for documentation. However markdown format is preferred than text format. please check.

Reply:
Done